packages/salesforcedx-vscode-apex/test/languageServer.test.ts

```diff
@@ -16,7 +16,10 @@ describe('Apex Language Server Client', () => {
     let originalPlatform: PropertyDescriptor;
 
     before(() => {
-      originalPlatform = Object.getOwnPropertyDescriptor(process, 'platform');
+      originalPlatform = Object.getOwnPropertyDescriptor(
```

Comment:
I think this is a combination of https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#weak-type-detection and because `PropertyDescriptor` has all optional fields and could return undefined.

Reply:
I could also use the non-null assertion operator here instead of casting. See https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator

I think I will switch to that since the intent is clearer. Using then non-null assertion operator seems to be more intention-revealing here.